.github/workflows/ci.yml

```
@@ -1,38 +1,28 @@
     runs-on: ubuntu-latest
-    permissions:
-      contents: read
-    env:
-      CARGO_TERM_COLOR: always
-      BUILD_PROFILE: debug
     steps:
       - uses: actions/checkout@v4
-      - name: Setup Rust
-        uses: leynos/shared-actions/.github/actions/setup-rust@v1.1.0
-      - name: Format
+      - uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)